pysynphot/__init__.py

```diff
@@ -12,43 +12,65 @@
 __svn_version__ = __svn_revision__
 __full_svn_info__ = __svn_full_info__
 
+def _init():
+    #UI:
+    #AnalyticSpectra:
+    from .spectrum import BlackBody, GaussianSource, FlatSpectrum
+    from .spectrum import Powerlaw as PowerLaw
+    #Tabular Spectra
+    from .spectrum import FileSourceSpectrum as FileSpectrum
+    from .spectrum import ArraySourceSpectrum as ArraySpectrum
+    from .catalog import Icat
+    #Analytic Spectral Elements
+    from .spectrum import Box, UniformTransmission
+    #Tabular Spectral Elements
+    from .spectrum import FileSpectralElement as FileBandpass
+    from .spectrum import ArraySpectralElement as ArrayBandpass
+    #Complicated spectral elements
+    from .obsbandpass import ObsBandpass
+    from .reddening import Extinction
+    #Observations
+    from .observation import Observation
+    #Other constructs
+    from .observationmode import ObservationMode as Obsmode
+    from numpy import arange as Waveset
+    #Get Vega
+    from .spectrum import Vega
+    #Get cache
+    from . import Cache
+    #Permit resetting refdata
+    from .refs import setref, showref
+    #
+    from .locations import get_data_filename
+    from . import tables
+    globals().update(locals())
 
-#UI:
-#AnalyticSpectra:
-from .spectrum import BlackBody, GaussianSource, FlatSpectrum
-from .spectrum import Powerlaw as PowerLaw
-#Tabular Spectra
-from .spectrum import FileSourceSpectrum as FileSpectrum
-from .spectrum import ArraySourceSpectrum as ArraySpectrum
-from .catalog import Icat
-#Analytic Spectral Elements
-from .spectrum import Box, UniformTransmission
-#Tabular Spectral Elements
-from .spectrum import FileSpectralElement as FileBandpass
-from .spectrum import ArraySpectralElement as ArrayBandpass
-#Complicated spectral elements
-from .obsbandpass import ObsBandpass
-from .reddening import Extinction
-#Observations
-from .observation import Observation
-#Other constructs
-from .observationmode import ObservationMode as Obsmode
-from numpy import arange as Waveset
-#Get Vega
-from .spectrum import Vega
-#Get cache
-from . import Cache
-#Permit resetting refdata
-from .refs import setref, showref
-#
-from .locations import get_data_filename
-from . import tables
+user_path = None
+def use(path):
+    import pysynphot
+    pysynphot.user_path = path
+    pysynphot._init()
+    return None
+
+def _root():
```

Comment:
Moving this to `locations.py` and use it to set `locations.rootdir` would make more sense to me.